core/src/main/scala/org/bykn/bosatsu/NameKind.scala

```diff
@@ -2,38 +2,38 @@ package org.bykn.bosatsu
 
 import Identifier.Bindable
 
-sealed abstract class NameKind
+sealed abstract class NameKind[T]
```

Comment:
the changes in this file are a good change and can be PR'ed and merged independently. I like to look out for anything that is small and easy to merge and understand atomically.